en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -9,41 +9,27 @@ iconBrand: js
 The following document is an advanced guide on building rich javascript interactions within the Silverstripe CMS and
 a list of our best practices for contributing and modifying the core javascript framework.
 
-## ES6 and build tools
```

Comment:
This isn't really relevant for this section of docs, so it was easier to remove it than to update it.